common/faceting.js

```diff
@@ -966,6 +1039,12 @@
                         params.faceting = false;
                         params.facetPanelOpen = false;
 
+                        // callback on each selected change (incldues the url limitation logic)
+                        params.onRowClick = modalDataChanged(scope, false);
+
+                        // if url limitation alert exists, remove it.
+                        AlertsService.deleteURLLimitAlert();
```

Comment:
Is this done because you can hit the length limitation on recordset with faceting, which will show the alert?

And then if you click "Show details ..." it would still show that alert, even though the multi-select in the popup can have the same issue?

Reply:
If you hit the limit in the recordset, we will not allow you to perform that action. Assume the url length is 1950 and selecting one of the facets will increase the length to more than the limit. When you click on the check box, we will show the alert and uncheck the checkbox. So you cannot perform that action at all. But in the modal, you can choose as much filters as you want. We won't prevent you from selecting filters. Only the submit button will be disabled.

Therefore, if there's an alert on the main page, that was because you wanted to do something and we prevented you from doing it. We don't need to keep showing you the alert. So when you open the modal, we can remove that alert.